mirth.rb

```diff
@@ -18,23 +18,22 @@ def all_paths
   end
 
   def show_data
-    response_body = "<ul>\n"
-    DailyData.all.each do |daily_data|
-      response_body += "<li> On this day <b>#{CGI.escapeHTML(daily_data.date)}</b>, #{CGI.escapeHTML(daily_data.step_count.to_s)}, #{CGI.escapeHTML(daily_data.notes)}</li>\n"
-    end
+    template = <<~HERE
+      <ul><% DailyData.all.each do |daily_data| %> 
```

Comment:
In general it’s in our best interests to keep as much code outside of the template as possible. Is there a way you could call `DailyData.all` outside of the template and use its value here?

Reply:
Do you mean just the `DailyData.all` part? I can assign that to a local variable and call that instead.

Reply:
Yes, just the `DailyData.all`. But to do this you have to figure out how you refer to a variable in a controller action from inside a template.

Reply:
Added in this commit [`cf97437` (#11)](https://github.com/wildmaples/mirth/pull/11/commits/cf974372cce6b2e57fcc2cfa2c5755fe2b8c82b8)